app/views/shared/authentication/_providers_registration_form.html.erb

```diff
@@ -8,7 +8,7 @@
       <%= params[:state] == "new-user" ? "Sign up" : "Continue" %> with <%= provider.official_name %>
     </a>
   <% end %>
-  <% if params[:state] == "new-user" && SiteConfig.allow_email_password_registration %>
+  <% if params[:state] == "new-user" && SiteConfig.allow_email_password_registration && !SiteConfig.invite_only_mode %>
```

Comment:
While QAing my work, I noticed that when `invite-only mode` was turned on, the Sign Up page still had the Sign up with Email option. So I updated this if-clause to correct this.